plugins/funind/g_indfun.mlg

```diff
@@ -47,15 +47,15 @@ let pr_fun_ind_using_typed prc prlc _ opt_c =
       let env = Global.env () in
       let evd = Evd.from_env env in
       let (_, b) = b env evd in
-      spc () ++ hov 2 (str "using" ++ spc () ++ Miscprint.pr_with_bindings prc prlc b)
+      spc () ++ hov 2 (str "using" ++ spc () ++ Miscprint.pr_with_bindings (prc env evd) (prlc env evd) b)
 
 }
 
 ARGUMENT EXTEND fun_ind_using
   TYPED AS constr_with_bindings option
   PRINTED BY { pr_fun_ind_using_typed }
-  RAW_PRINTED BY { pr_fun_ind_using }
-  GLOB_PRINTED BY { pr_fun_ind_using }
+  RAW_PRINTED BY { pr_fun_ind_using env sigma }
+  GLOB_PRINTED BY { pr_fun_ind_using env sigma }
```

Comment:
I am not sure I am very fond of this kind of hidden binding; I'd rather have the form take a function.

Reply:
cc @ppedrot

Reply:
This is for compatibility. With the current approach, only term-related printers need to be changed.

Reply:
Indeed, OK with that then.